launchable/commands/subset.py

```diff
@@ -85,7 +85,7 @@ def __init__(self):
             self.test_paths = []
             # TODO: robustness improvement.
             self._formatter = Optimize.default_formatter
-            self._separator = "\n"
+            self._separator = " "
```

Comment:
Is this change safe? At a casual glance, this looks like it's going to impact every test runner support

Reply:
Yeah, this is what I worried about. So, I'm going to modify it to limit to the file plugin instead of `subset.py`. Then, If the file plugin is okay at all customers environments, I will change default operator to white space, like a staged rollout.